### DISCUSSION TITLE
Updating multiple requests at the same time

### DISCUSSION THREAD

**Discussion by seven-deuce** [Kong/insomnia#4008] [Ideas]
Hi guys
So I have many endpoints in one folder, they need some authorization header to work.
Now updating Authorization header for each of them is a pain in the imagination. Is it possible to them all at once? 
If not, it would be great if you can let us select multiple requests and update headers of it.

By the way, insomnia is great and I am not going back to postman days!

**Comment by Spider101** [comment]
I think there is a plugin to set a global level header(s) for all requests in a collection - https://insomnia.rest/plugins/insomnia-plugin-global-headers. I don't think there is any native support for this though!
